packages/security_detection_engine/kibana/security_rule/68994a6c-c7ba-4e82-b476-26a26877adf6.json

```
@@ -51,7 +51,7 @@
         ],
         "timestamp_override": "event.ingested",
         "type": "query",
-        "version": 11
+        "version": 10
```

```suggestion
        "version": 12
```

Bumping as a result of bug identified: https://github.com/elastic/detection-rules/issues/1949